sabakan/generator.go

```diff
@@ -35,7 +37,16 @@ func MachineToNode(m *Machine, tmpl *cke.Node) *cke.Node {
 	}
 
 	for k, v := range tmpl.Annotations {
-		n.Annotations[k] = v
+		t, err := template.New(k).Parse(v)
```

Comment:
Does this keep compatibility from the original implementation?
Any valid `v` in the previous implementation can be parsed as Go template??